Library/Formula/freeswitch.rb

```diff
@@ -138,6 +140,7 @@ class Freeswitch < Formula
   #------------------------ End sound file resources --------------------------
 
   def install
+    ENV.deparallelize
```

Comment:
Including a link to that is fine then. Just something that shows it has ended up upstream and hopefully won't stick around in-formula indefinitely.